### PR TITLE
Removed old version code

### PR DESCRIPTION
**Pull request by michaelgugino**
This commit removes any references to versions < 1.5/3.5

We assume the version is always greater than or equal to 1.5/3.5.

### CODE REVIEW

File: roles/openshift_master/templates/master.yaml.v1.j2 [hunk 1]
```diff
@@ -159,31 +133,22 @@ kubernetesMasterConfig:
...
     contentType: application/vnd.kubernetes.protobuf
     burst: {{ openshift_master_loopback_ratelimit_burst | default(600) }}
     qps: {{ openshift_master_loopback_ratelimit_qps | default(300) }}
-{% endif %}
   openshiftLoopbackKubeConfig: openshift-master.kubeconfig
 masterPublicURL: {{ openshift.master.public_api_url }}
 networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
-{% if openshift.common.version_gte_3_7 | bool %}
```
Comment (mtnbikenc) [inline roles/openshift_master/templates/master.yaml.v1.j2]: Was this removal intentional?

File: roles/openshift_master/templates/master.yaml.v1.j2 [hunk 2]
```diff
@@ -281,4 +238,4 @@ servingInfo:
 {% endfor %}
 {% endif %}
 volumeConfig:
-  dynamicProvisioningEnabled: {{ openshift.master.dynamic_provisioning_enabled }}
+dynamicProvisioningEnabled: {{ openshift.master.dynamic_provisioning_enabled }}
```
Comment (mtnbikenc) [inline roles/openshift_master/templates/master.yaml.v1.j2]: Was this whitespace change intentional?

File: roles/openshift_master/templates/master.yaml.v1.j2 (hunk 1 above)
Comment (michaelgugino) [reply roles/openshift_master/templates/master.yaml.v1.j2]: Hmm... probably not, good catch.

File: roles/openshift_master/templates/master.yaml.v1.j2 (hunk 2 above)
Comment (michaelgugino) [reply roles/openshift_master/templates/master.yaml.v1.j2]: Hmm, probably not.  Good catch.

**Comment by michaelgugino** [comment]
bot, retest this please

**Comment by michaelgugino** [comment]
tox test received a sigquit for some reason...

/test tox

**Comment by sdodson** [comment]
thats'a f alek
